components/org.wso2.carbon.identity.oauth2.grant.organizationswitch/src/main/java/org/wso2/carbon/identity/oauth2/grant/organizationswitch/OrganizationSwitchGrant.java

```diff
@@ -127,6 +137,7 @@ public boolean validateGrant(OAuthTokenReqMessageContext tokReqMsgCtx) throws Id
 
         String[] allowedScopes = tokReqMsgCtx.getOauth2AccessTokenReqDTO().getScope();
         tokReqMsgCtx.setScope(allowedScopes);
+        tokReqMsgCtx.addProperty("tokenBindingReference", tokenDO.getTokenBinding().getBindingReference());
```

Comment:
Shall we extract "tokenBindingReference" to a constant?

Reply:
Fixed with [30583f1](https://github.com/wso2-extensions/identity-oauth2-grant-organization-switch/pull/12/commits/30583f1da23c808e776138e060865e40b93b7b55)